packages/frontend/src/app/(sticky-header)/about/page.tsx

```diff
@@ -152,7 +154,12 @@ export default async function About() {
       const avatar = res?.data?.data?.author?.avatar?.imageFile?.url
       member.avatar = avatar ? `${CMS_URL}${avatar}` : DEFAULT_AVATAR
```

Comment:
Do we need to check if `member` is an object before requesting API ?
If `member` is `null` or `undefined`, `member.slug` will throw error, 
and then the error will not be the axios error that `errors.helpers.annnotateAxiosError` could handle.

Reply:
member is an element of teamMember(typed constant), so undefined/null will not be allowed after type check